source/github-helpers/prevent-link-loss.ts

```diff
@@ -1,40 +1,43 @@
+import {getRepo} from '.';
+
+const currentRepo = getRepo() ?? {nameWithOwner: 'refined-github/refined-github'};
```

Comment:
The fallback is for tests only. I couldn't find how to mock stuff in AVA, is there a better way than this?

Reply:
> is there a better way than this?

Probably! But if you don't want to look into it it's fine as is